github_jira/verbiages/verbiages.go

```diff
@@ -0,0 +1,10 @@
+package verbiages
```

Comment:
I think this package name is hard to understand for not-native speakers, It is not a really common word to use, so I would go for something like `templates`. Also, because we only have 1 template and is used as part of our main package, I don't see the need of this now. I would create a `templates.go` file in the `github` package with this, and thats it.